### DISCUSSION TITLE
Is there a recommended setup for use in an existing React app?

### DISCUSSION THREAD

**Discussion by ogrotten** [tailwindlabs/tailwindcss#2891] [Help] [answered]
Google responds with _several_ different methods to installing tailwindcss for use with react, and all of them use a different set of additional modules.

Is there a recommended or preferred/sanctioned way to setup to an existing React app?

**Comment by adamwathan** [comment]
What specific type of set up are you using? Next.js, create react app, custom webpack config?

**Comment by ogrotten** [comment]
Let's start with create-react-app.

More to the point is the question is there documentation of it? I'm guessing there is a preferred method of install, but is there an install guide or wiki page or the like for those options out there somewhere?

**Comment by simonswiss** [comment, marked as answer]
Hey!

There are now integration guides, and one of them covers Create React App 👍

https://tailwindcss.com/docs/guides/create-react-app
